roles/nova-data/tasks/libvirt.yml

```diff
@@ -56,29 +58,157 @@
   notify: restart nova services
   when: openstack_install_method == 'source'
 
+- name: make pki paths
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: kvm
+  with_items:
+    - /etc/pki/CA
+    - /etc/pki/libvirt/private
+
+- name: get uname -n
+  command: uname -n
+  register: uname
+
+- name: create 'uname -n' as fact 'uname_host'
+  set_fact:
+    uname_host: "{{ uname.stdout }}"
+
+- name: install CA file
+  template:
+    src: cacert.pem.j2
```

Comment:
source path should match as much as it can where it'll be on the remote host.  So in this case, it should be `etc/pki/CA/cacert.pem`